<launch>
    <rosparam command="load" file="$(find witmotion_ros)/config/wt31n.yml" />
    <node name="witmotion_imu" pkg="witmotion_ros"  type="witmotion_ros_node" output="screen"/>
</launch>
